src/mode.c

```diff
@@ -52,8 +56,8 @@ bool check_mode(uint32_t now)
       // if left stick is down and to the right
       if (pwmRead(_params.values[PARAM_RC_F_CHANNEL]) < _params.values[PARAM_RC_F_BOTTOM] +
           _params.values[PARAM_ARM_THRESHOLD]
-          && pwmRead(_params.values[PARAM_RC_Z_CHANNEL]) > (_params.values[PARAM_RC_Z_CENTER] + _params.values[PARAM_RC_Z_RANGE]/2)
-          - _params.values[PARAM_ARM_THRESHOLD])
+          && pwmRead(_params.values[PARAM_RC_Z_CHANNEL]) < (_params.values[PARAM_RC_Z_CENTER] - _params.values[PARAM_RC_Z_RANGE]/2)
```

Comment:
I think we've already got an issue for this, but it'd be nice to have a separate RC module that handles all of the calibration stuff and just returns normalized values between -1 and 1 or 0 and 1 for each channel. But again, let's not worry about it here.